tools/wpt/browser.py

```diff
@@ -548,22 +548,44 @@ def _official_chromedriver_url(self, chrome_version):
             latest, self.platform_string())
 
     def _chromium_chromedriver_url(self, chrome_version):
-        try:
-            # Try to find the Chromium build with the same revision.
-            omaha = get("https://omahaproxy.appspot.com/deps.json?version=" + chrome_version).json()
-            revision = omaha['chromium_base_position']
-            url = "https://storage.googleapis.com/chromium-browser-snapshots/%s/%s/chromedriver_%s.zip" % (
-                self.chromium_platform_string(), revision, self.platform_string())
-            # Check the status without downloading the content (this is a streaming request).
-            get(url)
-        except requests.RequestException:
-            # Fall back to the tip-of-tree Chromium build.
-            revision_url = "https://storage.googleapis.com/chromium-browser-snapshots/%s/LAST_CHANGE" % (
-                self.chromium_platform_string())
+        # The algorithm for downloading Chromedriver is documented at
+        # https://chromedriver.chromium.org/downloads/version-selection
```

Comment:
This is in fact `_official_chromedriver_url`. 

We should instead rework `_latest_chromedriver_url` to make it try `_official_chromedriver_url(current_release)`, `_official_chromedriver_url(current_release-1)`, and then finally `_chromium_chromedriver_url(current_release)`.